### ISSUE TITLE
Cannot setup 2.3.2 on Windows 'Zend\Mvc\Controller\LazyControllerAbstractFactory' not found

### ISSUE THREAD

**Issue by kAleksei** [magento/magento2#30499]
### Preconditions (*)

1. First install of magento 2.3.2
2. Windows 10 (2004)
3. PHP 7.2.34

### Steps to reproduce (*)

1. Run composer install
2. Run bin/magento setup:upgrade

### Expected result (*)

1. Setup/upgrade should run 

### Actual result (*)

1. After running bin/magento setup:upgrade in PowerShell as Administrator
![image](https://user-images.githubusercontent.com/21226411/96114946-ed3aef80-0eee-11eb-9772-d22f23806e6a.png)
2. composer.json
``` 
{
  "name": "magento/project-community-edition",
  "description": "eCommerce Platform for Growth (Community Edition)",
  "type": "project",
  "version": "2.3.2",
  "license": [
    "OSL-3.0",
    "AFL-3.0"
  ],
  "require": {
    "composer/composer": "@alpha",
    "sbodak/magento2-b2b-disable-customer-registration": "^1.0",
    "magento/product-community-edition": "2.3.2",
    "mageplaza/module-smtp": "^1.3"
  },
  "require-dev": {
    "allure-framework/allure-phpunit": "~1.2.0",
    "friendsofphp/php-cs-fixer": "~2.13.0",
    "lusitanian/oauth": "~0.8.10",
    "magento/magento-coding-standard": "~1.0.0",
    "magento/magento2-functional-testing-framework": "~2.3.14",
    "pdepend/pdepend": "2.5.2",
    "phpmd/phpmd": "@stable",
    "phpunit/phpunit": "~6.5.0",
    "sebastian/phpcpd": "~3.0.0",
    "squizlabs/php_codesniffer": "3.3.1"
  },
  "autoload": {
    "psr-4": {
      "Magento\\Framework\\": "lib/internal/Magento/Framework/",
      "Magento\\Setup\\": "setup/src/Magento/Setup/",
      "Magento\\": "app/code/Magento/",
      "Zend\\Mvc\\Controller\\": "setup/src/Zend/Mvc/Controller/"
    },
    "psr-0": {
      "": [
        "app/code/",
        "generated/code/"
      ]
    },
    "files": [
      "app/etc/NonComposerComponentRegistration.php"
    ],
    "exclude-from-classmap": [
      "**/dev/**",
      "**/update/**",
      "**/Test/**"
    ]
  },
  "autoload-dev": {
    "psr-4": {
      "Magento\\Sniffs\\": "dev/tests/static/framework/Magento/Sniffs/",
      "Magento\\Tools\\": "dev/tools/Magento/Tools/",
      "Magento\\Tools\\Sanity\\": "dev/build/publication/sanity/Magento/Tools/Sanity/",
      "Magento\\TestFramework\\Inspection\\": "dev/tests/static/framework/Magento/TestFramework/Inspection/",
      "Magento\\TestFramework\\Utility\\": "dev/tests/static/framework/Magento/TestFramework/Utility/"
    }
  },
  "minimum-stability": "stable",
  "repositories": [{
    "type": "composer",
    "url": "https://repo.magento.com/"
  }],
  "extra": {
    "magento-force": "override"
  }
}
```

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [x] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by kAleksei** [comment]
The solution from this issue didn't help https://github.com/magento/magento2/issues/15441

**Comment by sdzhepa** [comment]
Hello @kAleksei 

Thank you for report and collaboration.

Unfortunately, Magento does not support Windows
Quote from [Magento 2.4 technology stack requirements](https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html#operating-systems-linux-x86-64)
> Linux distributions, such as RedHat Enterprise Linux (RHEL), CentOS, Ubuntu, Debian, and similar. **Magento is not supported on Microsoft Windows and macOS.**

Also, in this repo we can collaborate/fix/deliver only issues related to 2.4.x and that can be reproduced on the latest `2.4-develop` code base.

So, if you have steps to reproduce on the supported OS and version please update the description.
In this case, we will be able to confirm/triage this issue and move to the "Ready for Dev" status

**Comment by kAleksei** [comment]
No. I can't reproduce this issue on supported OS. So I`ll close it.
